07-routing-gossip.md

```diff
@@ -80,13 +80,13 @@ announcement message; that is done by having a signature from each
     * [`64`:`node_signature_2`]
     * [`64`:`bitcoin_signature_1`]
     * [`64`:`bitcoin_signature_2`]
+    * [`2`:`len`]
```

Comment:
Fair point. We have a good bit of space in the type bytes that we can use to add additional messages in the future. In a sense, I guess it's also a bit more explicit to add in a new message type (keeping in the even/odd semantics) vs modifying some feature bits and creating a new message entirely.